kanidm_book/src/oauth2.md

```diff
@@ -195,15 +195,17 @@ or with an appropriate include.
     OIDCPKCEMethod S256
     OIDCCookieSameSite On
 
-Other scopes can be added as required to the `OIDCScope` line, eg: `OIDCScope "openid scope2 scope3"`
-
 In the virtual host, to protect a location:
 
     <Location />
         AuthType openid-connect
         Require valid-user
     </Location>
 
+Other scopes can be added as required to the `OIDCScope` line, eg: `OIDCScope "openid scope2 scope3"`
+
+If you want to set the `REMOTE_USER` field to the `preferred_username` instead of the UUID, add the line `OIDCRemoteUserClaim preferred_username`. Remember that the username can change, but this can help with systems like Nagios which use this as a display name.
```

Comment:
Why not have this as a commented example on the lines above?

Reply:
Updated